src/main/java/com/odysseusinc/arachne/portal/service/impl/BaseStudyServiceImpl.java

```diff
@@ -266,6 +269,18 @@ private void init() {
         }};
     }
 
+    @Override
+    public T createWorkspace(IUser owner, T workspace) {
+
+        StudyType studyType = new StudyType();
```

Comment:
studyType will be returned from service in next line, why do you want to create one more?